pkg/utils/config.go

```diff
@@ -31,6 +32,8 @@ type IDHubConfig struct {
 	PersisterPostgresMaxConns *int               `split_words:"true" desc:"If persister type is Postgresql, sets the max conns in pool"`
 	PersisterPostgresMaxIdle  *int               `split_words:"true" desc:"If persister type is Postgresql, sets the max idle conns in pool"`
 	PersisterPostgresConnLife *int               `split_words:"true" desc:"If persister type is Postgresql, sets the max conn lifetime in secs"`
+
+	RedisHosts []string `required:"true" split_words:"true" desc:"List of Redis host:port for caching and locking"`
```

Comment:
should this be optional and fall back to local lock?

Reply:
Will update so this config is not required and will fallback to local lock.